Dockerfile

```diff
@@ -21,7 +21,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/* \
     && ln -sf /dev/stdout /var/log/nginx/access.log \
     && ln -sf /dev/stderr /var/log/nginx/error.log \
-    && npm install -g npm@7.6.3
+    && npm install -g npm@7.8.0 \
+    && npm config set maxsockets 5
```

Comment:
can you also add this into Travis script? just to keep consistancy and prevent any more drama from happening in the future